elastic_apm.services.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  logger.channel.elastic_apm:
+    class: Drupal\Core\Logger\LoggerChannel
+    factory: logger.factory:get
+    arguments: ['elastic_apm']
+
+  http_middleware.elastic_apm:
+    class: Drupal\elastic_apm\StackMiddleware\ElasticApmMiddleware
+    tags:
+      - { name: http_middleware, priority: 350 }
+
+  elastic_apm.service:
+    class: Drupal\elastic_apm\ElasticApm
```

Comment:
**CHANGE REQUEST**

Let's give it a better name, ElasticApm means nothing. `ApiService`, as it is related to configuring and getting the API agent?